quickstarts/OWNERS

```diff
@@ -0,0 +1,18 @@
+reviewers:
+  - invincibleJai
+  - jerolimov
+  - jhadvig
+  - lokanandaprabhu
+  - lucifergene
+  - serenamarie125
+  - spadgett
+  - vikram-raj
+approvers:
+  - invincibleJai
+  - jerolimov
+  - jhadvig
+  - serenamarie125
+  - spadgett
+  - vikram-raj
+labels:
+  - component/quickstarts
```

Comment:
Can we add any labels here or is it required that the project contains this label already?

Reply:
@jerolimov not sure what labels you mean.

Also could you please add @TheRealJon ? Thanks!

Reply:
Doesn't this label get automatically added to all PRs that touch a file in the quickstart?

Sure, I added him

Reply:
>Doesn't this label get automatically added to all PRs that touch a file in the quickstarts folder?

it does

Reply:
Since @TheRealJon is an approver in the root OWNERS file, he can already approve quick start changes and doesn't need to be specifically in this file.

I believe labels need to be added to the release repo as in https://github.com/openshift/release/pull/14656

Reply:
I would repeat him if he has been a delegate or responsible in the past for quickstart, not when its just a fallback like you both. The idea is to streamline this and reduce the work for you both. :smirk: 

For now, I dropped him again and removed the label as it's unimportant, with just 1-2 PRs per release.

I keep the editor open if I need to change something else :laughing: